content/en/database_monitoring/agent_integration_overhead.md

```diff
@@ -0,0 +1,93 @@
+---
+title: DBM Agent Integration Overhead
+kind: documentation
+description: Learn about the overhead of integrating the Datadog Agent with your database
+
+---
+
+## Overview
+
+Database Monitoring runs on top of the base Datadog Agent. By default, it's configured to perform optimally while ensuring minimal impact on your system. However, you have the flexibility to adjust settings like data collection frequency and query sampling to better suit your needs.
+
+For most workloads, the Agent's resource usage remains minimal, accounting for less than one percent of query execution time and CPU usage on your database.
```

Comment:
Talked to @caseycull and I thought we were going to drop this to focus on the factual case of performance benchmarks?

Reply:
Just drop `For most workloads, the Agent's resource usage remains minimal, accounting for less than one percent of query execution time and CPU usage on your database.` right? not drop everything under `Overview`?